src/NiceWrapper.cpp

```diff
@@ -257,11 +282,7 @@ void NiceWrapper::SendLoop() {
   }
 }
 
-void NiceWrapper::SetRemoteCredentials(std::string username, std::string password) {

```

Comment:
Just so I'm clear - now we parse the entire SDP, and rely on libnice parsing to extract credentials?

If I remember correctly, there is a drawback to this approach, but I can't remember quite what it is right now.
This is great for now, but lets review in future.